### PR TITLE
Gpl 913 housekeeping remove cgap integration

### LINKED ISSUE #236: GPL-913 Housekeeping - remove CGaP integration
**Describe the Housekeeping**
The slack exception channel keeps getting messages when barcodes are scanned into asset audits which then calls the wrangler to search for these barcodes in the MLWH table which CGaP used to populate. 

This functionality is no longer used nor required.

**Blocking issues**
Describe any other issues or tickets that may be blocking this change.

**Additional context**

### PR DESCRIPTION
**Pull request by stevieing**
Closes #236 

Changes proposed in this pull request:

*
*
* ...

### CODE REVIEW

**Review by pjvv — APPROVED** [review]
There would also be a related deployment change now that I think about it.

Want to include some gem updates?

**Comment by stevieing** [comment]
> There would also be a related deployment change now that I think about it.
> 
> Want to include some gem updates?

I'll get a few of them done before stand up.

File: app/controllers/process_plates_controller.rb [hunk 1]
```diff
@@ -36,9 +34,6 @@ def create
     # 20 barcodes takes about 2 mins as of 2020-06-22. This limit is to prevent the request timing out.
     raise "Please scan #{ProcessPlate::RECEIVE_PLATES_MAX} barcodes or fewer" if barcodes.count > ProcessPlate::RECEIVE_PLATES_MAX
```
Comment (JamesGlover) [inline app/controllers/process_plates_controller.rb:36]: This limit was added due to wrangler performance issues, so can also be removed.

File: app/controllers/process_plates_controller.rb [hunk 2]
```diff
@@ -36,9 +34,6 @@ def create
...
 
-    responses = call_external_services(barcodes)
-    @results = generate_results(barcodes, responses)
-
```
Comment (JamesGlover) [inline app/controllers/process_plates_controller.rb:41]: You're no longer setting @results, but are rendering the results template, which contains:

```haml
# app/views/process_plates/results.html.haml
  %tbody
    - @results.each do |barcode, details|
```

Not sure why this isn't blowing up, so sorry if I missed something. Still, probably makes sense to remove the results render, as the table is always going to be empty.

It used to just:
```
flash[:notice] = 'success'
...
format.html { redirect_to(new_process_plate_path) }
```

Although the new flash (which should be flash.now for a render) definitely looks like an improvement.

**Review by JamesGlover — CHANGES_REQUESTED** [review]
Couple of potential issues I spotted. I'm not sure why the template problem isn't a noisy one.
The limit can definitely be removed though.

**Review by JamesGlover — APPROVED** [review]

**Pull request merged**